great_expectations/profile/base.py

```diff
@@ -144,20 +144,23 @@ class Profiler(metaclass=abc.ABCMeta):
       kind of object. You should raise an appropriate Exception if the object is not valid.
     """
 
-    def __init__(self, configuration: dict = None) -> None:
+    def __init__(self, configuration: Optional[dict] = None) -> None:
         self.configuration = configuration
 
-    def validate(self, item_to_validate: Any) -> None:
-        pass
+    def validate(self, item_to_validate: Any) -> bool:
+        # TODO: Re-evaluate on this behavior
+        return False
```

Comment:
I'm going to revert this change and just ignore the related type-error along with a comment.